schedule/__init__.py

```diff
@@ -48,11 +48,22 @@
 logger = logging.getLogger('schedule')
 
 
+class ScheduleError(Exception):
+    """Base schedule exception"""
+
+
+class ScheduleValueError(ScheduleError):
+    """Base schedule value error"""
+
+
+class IntervalError(ScheduleValueError):
+    """An improper interval was used"""
+
+
 class CancelJob(object):
     """
     Can be returned from a job to unschedule itself.
     """
-    pass
```

Comment:
What happened here? Let's leave the `pass` in to make sure we communicate that this should be an empty class. (Same above on the various error classes, just a stylistic thing)